client/log_client.go

```diff
@@ -357,6 +357,15 @@ func (c *LogClient) AddSequencedLeaf(ctx context.Context, data []byte, index int
 	return err
 }
 
+// AddSequencedLeaves adds any number of pre-sequenced leaves to the log.
+func (c *LogClient) AddSequencedLeaves(ctx context.Context, leaves []*trillian.LogLeaf) error {
```

Comment:
For consistency and ease of use, it might be better for this to work like `AddSequencedLeaf()`, e.g.
```go
func (c *LogClient) AddSequencedLeaves(ctx context.Context, data map[int64][]byte) error {
	leaves := make([]*LogLeaf, 0, len(data))
	for i, d := range data {
		leaf, err := c.BuildLeaf(d)
		if err != nil {
			return fmt.Errorf("error building leaf %d: %v", i, err)
		}
		leaf.LeafIndex = i
		leaves = append(leaves, leaf)
	}
	_, err := c.client.AddSequencedLeaves(ctx, &trillian.AddSequencedLeavesRequest{
		LogId:  c.LogID,
		Leaves: leaves,
	})
	return err
}
```

Reply:
amazing idea.  Done.